src/ec/packages/ec-component-inpage-navigation/inpage-navigation.story.js

```diff
@@ -33,14 +33,6 @@ storiesOf('Components/Navigation/Inpage navigation', module)
       htmlElement.innerHTML = `<div class="ecl-container"><div class="ecl-row ecl-u-mt-l"><div class="ecl-col-md-3">${html}</div><div class="ecl-col-md-9">${pageFillerHtml}</div></div></div>`;
       demo.appendChild(htmlElement.firstChild);
 
```

Comment:
Also here we don't have knobs, let's cleanup, then...

Reply:
cleaned.